docs/EVM/developer-tooling.md

```diff
@@ -129,3 +129,19 @@ module.exports = {
 ```
 
 Deploy/Migrate by running `truffle migrate --network shibuya`, replacing `shibuya` with your chosen network. If `--network` is not specified, the network values under`development` will be used.
+
+## Your own RPC server
+
+For EVM developers and projects could be reasonable to have own EVM endpoints. In case of using public endpoints the project will depend of this endpoint, centralized EVM RPC can become single point of failure. Officially Astar team recommend projects to use their own managed EVM endpoints.
+
+Fortunately launch Astar Network endpoint is not that difficult.
+
+> By default EVM RPC server is disabled, for turn it on please add `--enable-evm-rpc` flag into launch string.
```

Comment:
```suggestion
:::note
By default EVM RPC server is disabled, to turn it on please add `--enable-evm-rpc` flag into the launch string.
:::
```

I think you should mention this in the chapter which describes operating a node in general.